main.js

```diff
@@ -16,7 +16,6 @@ const months = JSON.parse('["01_January","02_February","03_March","04_April","05
 // Create variables for object arrays
 const history    = {};
 const aliasMap   = {};
-const wh_start_val = [];
```

Comment:
unused

Reply:
not yet used, working an implementing it were this variable is needed